services/ui-src/src/utils/tableListExportToCSV.js

```diff
@@ -1,9 +1,11 @@
-import { roleLabels } from "cmscommonlib";
+import { roleLabels, ChangeRequest } from "cmscommonlib";
 import { format } from "date-fns";
 
 const CSV_HEADER = {
   "submission-table":
     "SPA ID/Waiver Number,Type,State,Date Submitted,Submitted By",
+  "package-dashboard":
+    "SPA ID/Waiver Number,Type,State,Date Submitted,Submitted By",
```

Comment:
Probably not the domain of this story given it seems both exports were previously sharing CSV headers, but shouldn't the export contain all items from the view? IE package view also has "status" and "90th day" on the UI.